auth-web/src/store/modules/business.ts

```diff
@@ -154,6 +154,42 @@ export default class BusinessModule extends VuexModule {
       }
       affiliatedEntities.push(entity)
     })
+
+    const resp = await OrgService.getAffiliationInvitations(this.currentOrganization.id)
+      .catch(err => {
+        console.log(err) // eslint-disable-line no-console
+        return null
+      })
+
+    if (resp?.data?.affiliationInvites) {
+      const affiliationInviteInfos: AffiliationInviteInfo[] = resp.data.affiliationInvites
+
+      affiliationInviteInfos.forEach(affiliationInviteInfo => {
+        const business: Business = affiliatedEntities.find(businesses => businesses.businessIdentifier === affiliationInviteInfo.business.businessIdentifier)
+        const affiliationInviteInfosArray = [affiliationInviteInfo]
+
+        if (business) {
+          business.affiliationInvites = (business.affiliationInvites || []).concat(affiliationInviteInfosArray)
```

Comment:
Much cleaner looking, thank you for this

Reply:
agreed, and I appreciate comments that push for that kind of improvements!